<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QMainWindow" name="Termometro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/imagenes/recursos/termometro.png</normaloff>:/imagenes/recursos/termometro.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Centigrados</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLCDNumber" name="outCent">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 0);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdCent">
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Fahrenheit</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLCDNumber" name="outFahr">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 170, 0);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
           <property name="value" stdset="0">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdFahr">
           <property name="minimum">
            <number>-23</number>
           </property>
           <property name="maximum">
            <number>212</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Kelvin</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLCDNumber" name="outKel">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(25, 255, 247);
color: rgb(0, 0, 0);</string>
           </property>
           <property name="segmentStyle">
            <enum>QLCDNumber::Flat</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDial" name="cmdKel">
           <property name="minimum">
            <number>243</number>
           </property>
           <property name="maximum">
            <number>374</number>
           </property>
           <property name="value">
            <number>243</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTermometro">
    <property name="title">
     <string>Termometro</string>
    </property>
   </widget>
   <addaction name="menuTermometro"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cmdCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cmdKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>outKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
